.github/workflows/reusable-build-test-release.yml

```diff
@@ -2168,25 +2213,6 @@ jobs:
           path: |
             ${{ needs.setup.outputs.directory-path }}/diag*
 
-  validate-pr-title:
```

Comment:
Please provide reasons for removing this job.

Reply:
There are pros and cons for removing this job.

Pros of removing:

* It takes seconds to run, but it costs us 1 minute of billing time
* It is a miscellaneous job, totally optional. If a developer knows what they are doing, this job is not needed and I think we should expect this

Cons of removing:

* Enforces consistency for the PR title, but not for the commits

Reply:
- I think rather than removing this job we could move it to be a step in `setup` job, so it will not consume 1 minute of billing time.

- Sure we can expect that, but this can serve for example as a reminder in case someone forgot to add proper title.

- Enforcing consistency of PR title together with "Squash and merge" for PR results in clear overview of changes history, e.g.: https://github.com/splunk/splunk-add-on-for-cisco-meraki/commits/main

To summarize, I think following this action doesn't require significant human effort or computational resources if it's moved as a step to another job. Nonetheless, it provides value, so I prefer to retain it.

Reply:
Okay, I'll leave it, as a follow up let's have only "Squash and merge" option available for the add-ons. I'll take this on me.